modules/skybrowser/src/screenspaceskytarget.cpp

```diff
@@ -0,0 +1,430 @@
+ * OpenSpace                                                                             *
+ *                                                                                       *
+ * Copyright (c) 2014-2022                                                               *
+ *                                                                                       *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of this  *
+ * software and associated documentation files (the "Software"), to deal in the Software *
+ * without restriction, including without limitation the rights to use, copy, modify,    *
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software, and to    *
+ * permit persons to whom the Software is furnished to do so, subject to the following   *
+ * conditions:                                                                           *
+ *                                                                                       *
+ * The above copyright notice and this permission notice shall be included in all copies *
+ * or substantial portions of the Software.                                              *
+ *                                                                                       *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,   *
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A         *
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT    *
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+#include <modules/skybrowser/include/screenspaceskytarget.h>
+
+#include <modules/skybrowser/skybrowsermodule.h>
+#include <modules/skybrowser/include/screenspaceskybrowser.h>
+#include <modules/skybrowser/include/utility.h>
+#include <openspace/engine/globals.h>
+#include <openspace/engine/moduleengine.h>
+#include <openspace/navigation/navigationhandler.h>
+#include <openspace/rendering/renderengine.h>
+#include <openspace/rendering/helper.h>
+#include <openspace/scene/scene.h>
+#include <openspace/camera/camera.h>
+#include <ghoul/filesystem/filesystem.h>
+#include <ghoul/opengl/programobject.h>
+
+namespace {
+    constexpr const char* _loggerCat = "ScreenSpaceSkyTarget";
+
+    constexpr const std::array<const char*, 7> UniformNames = {
+        "modelTransform", "viewProj", "showCrosshair", "showRectangle", "lineWidth", 
+        "dimensions", "lineColor"
+    };
+
+    constexpr const openspace::properties::Property::PropertyInfo CrosshairThresholdInfo =
+    {
```

Comment:
The `{` should be moved to after the `=` on the previous line for all there PropertyInfos, as long as the line width don't exceed 90

Reply:
It will exceed 90 if i move it up :/

Reply:
But not for all the others, right? :)

Reply:
True:)